src/ImageService.Broker/Consumers/RemoveImagesConsumer.cs

```diff
@@ -0,0 +1,44 @@
+﻿using LT.DigitalOffice.ImageService.Data.Interfaces;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Models.Broker.Enums;
+using LT.DigitalOffice.Models.Broker.Requests.Image;
+using MassTransit;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ImageService.Broker.Consumers
+{
+    public class RemoveImagesConsumer : IConsumer<IRemoveImagesRequest>
+    {
+        private readonly IImageRepository _imageRepository;
```

Comment:
the consumer has one repository, you can rename variable to _repository

Reply:
done